tutorials/classical_simulation/Simulating with effective noise channels.ipynb

```diff
@@ -15,25 +15,23 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    noise_rates = np.round(np.linspace(0, 0.2, 5), 3)
</pre><p>So I guess we don't have to be restricted from 0 to 0.2 anymore?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
No we don't have to, but I don't think we would see much if going to higher rates.